webapp/components/wpt-flags.js

```diff
@@ -27,7 +27,9 @@ Object.defineProperty(wpt, 'ClientSideFeatures', {
     return [
       'colorHomepage',
       'diffFromAPI',
+      'edgeChromiumByDefault',
```

Comment:
I don't think this flag (and the corresponding branching) is necessary, because we will never have both EdgeHTML & EdgeChromium on the same channel, and this branching happens after the branching for channel preferences (`experimentalByDefault`). In other words, if `experimentalByDefault` is true, then we will only get EdgeChromium, so this flag becomes a no-op.

Even when we are looking at history runs, this flag is also a no-op. At no point in history have we ever received both EdgeHTML & EdgeChromium on the same channel, so the decision/timeframe decides which Edge you'll get.